network-api/networkapi/templates/fragments/language_switcher.html

```diff
@@ -1,15 +1,17 @@
-{% load i18n localization %}
+{% load i18n localization wagtailcore_tags %}
+
+{% get_current_language as current_language %}
+{% get_local_language_names as languages %}
 
 <form action="{% url 'set_language' %}" method="post">
-  <input name="next" type="hidden" value="{{ request.path }}">
+  {% csrf_token %}
```

Comment:
We'll want to figure out how to do this without CSRF to continue using CloudFront in production without running into caching issues.